user/enterprise/platform-tips.md

```diff
@@ -309,6 +309,16 @@ Additionally, please the following command to clean up all Docker build images:
 $ sudo docker images | grep travis | awk '{print $3}' | xargs sudo docker rmi -f
 ```
 
+## Integrate Travis CI Enterprise into your monitoring
+
+To implement a simple uptime check for Travis CI Enterprise, you can query the `/api/uptime` endpoint from your instance.
+
+```
+$ curl -H "Authorization: token XXXXX" https://travis.example.com/api/uptime
+```
+
+If everything is up and running, it answers with a `HTTP 200 OK`.
```

Comment:
Do we need to mention anything apart from a timeout here? Are there error codes?

Reply:
Good call! I checked [here](https://github.com/travis-ci/travis-api/blob/72fac430b2615e6f5ff7e3371f74bfa621ef59df/lib/travis/api/app/endpoint/uptime.rb) and we can either expect `200 OK` or a `500` in case of failure.